apps/cli/.eslintrc.js

```diff
@@ -3,23 +3,11 @@ module.exports = {
     node: true,
     commonjs: true,
     es2021: true,
+    jest: true,
   },
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-  ],
+  extends: ['eslint:recommended'],
   parserOptions: {
     ecmaVersion: 'latest',
   },
-  rules: {
-    '@typescript-eslint/no-unused-vars': [
-      'error',
-      {
-        ignoreRestSiblings: true,
-      },
-    ],
-    '@typescript-eslint/ban-ts-comment': 1,
-    '@typescript-eslint/no-var-requires': 1,
-  },
+  rules: {},
```

Comment:
there's no ts in `apps/cli`, removing all plugins from here

Reply:
we should use ts \o/